src/popup/router/components/Modals/QrCodeReader.vue

```diff
@@ -162,23 +162,25 @@ export default {
           ['body', '#app'].forEach((s) => {
             document.querySelector(s).style = 'background: transparent';
           });
-          document.querySelector('.main').style.display = 'none';
-          document.querySelector('.transfer-send-modal').style.display = 'none'; // TODO: refactor cordova qr code reading
-          this.$store.commit('setPageTitle', this.$t('modals.qrCodeReader.scanQr'));
+          document.querySelector('.app-inner').style.display = 'none';
+
+          const closeButton = document.querySelector('#camera-close');
+          closeButton.style.display = 'block';
```

Comment:
In my opinion here we should use the Vuex state and the close button should read the state to be displayed or not.